### PR TITLE
Fix scaling of mean moving average

### LINKED ISSUE #332: Unbiased estimation in batch normalization function
Hi!

I'm now implementing neural network models with chainer.
I have a question regarding in the implementation of batch normalization function in chainer.
Following code is the computation of the average of means and the average of varinaces for the test mode in chainer/functions/batch_normalization.py

            m = ldim * rdim
            adjust = m / max(m - 1., 1.)  # unbiased estimation
            self.avg_mean *= decay
            self.avg_mean += (1 - decay) * adjust * mean
            self.avg_var *= decay
            self.avg_var += (1 - decay) * adjust * var
Why is the estimation of the "avg_mean" scaled by m/(m-1)?
I think only the "avg_var" needs to be scaled by m/(m-1) for unbiased estimation.
In the original paper of batch normalization:http://arxiv.org/pdf/1502.03167v3.pdf, only the average of variances is also scaled by m/(m-1), and the average of means is not scaled. 

Best,

### PR DESCRIPTION
**Pull request by beam2d**
fix #332

### CODE REVIEW

**Comment by unnonouno** [comment]
LGTM
